frontend/eslint.config.js

```diff
@@ -0,0 +1,69 @@
+import react from "eslint-plugin-react"
+import hooks from "eslint-plugin-react-hooks"
+import reactRefresh from "eslint-plugin-react-refresh"
+import love from "eslint-config-love"
+import tsParser from "@typescript-eslint/parser"
+import globals from "globals"
+
+export default [
+  love,
+  {
+    rules: {
+      'react-refresh/only-export-components': 'warn',
+    }
+  },
+  {
+    files: [
+      '**/*.ts',
+      '**/*.tsx'
+    ],
+    rules: {
+      '@typescript-eslint/space-before-function-paren': [
+        'error',
+        {
+          anonymous: 'always',
+          named: 'never',
+          asyncArrow: 'always'
+        }
+      ],
+      'no-undef': 'off'
```

Comment:
Does this `no-undef` rule impact anything? (removing it doesn't seem to create errors)

Reply:
It does not - it is leftover from testing some features.